src/item/base-item.ts

```diff
@@ -0,0 +1,17 @@
+import type { ModifierTier } from "#enums/modifier-tier";
+
+interface SelectOption {
+  nameKey: string;
+  descKey: string;
+  price: number;
+  rarity: ModifierTier; // TODO: Rename this
+}
```

Comment:
```suggestion
interface SelectOption {
  price: number;
  rarity: ModifierTier; // TODO: Rename this
  
  get name(): string;
  get description(): string;
}
```
I figure the internal name/description keys can be abstracted from the UI code? You'd have to add the same methods to BaseItem, of course.

Reply:
Also iterate if any of those should/could be read only